### ISSUE TITLE
[flutter_tools] StateError: Bad state: No running isolate during flutter run -d chrome

### ISSUE THREAD

**Issue by christopherfujino** [flutter/flutter#121641]
on beta/3.8.0-10.1.pre
`flutter --no-color run --machine --track-widget-creation --device-id=chrome --start-paused --web-renderer html --dart-define=flutter.inspector.structuredErrors=true lib/main.dart`

This version is using dwds: 17.0.0 https://github.com/flutter/flutter/blob/beta/packages/flutter_tools/pubspec.yaml#L14

```
StateError: Bad state: No running isolate (inspector is not set).content_copy
at ChromeProxyService.inspector(chrome_proxy_service.dart:79)
at ChromeProxyService._prewarmExpressionCompilerCache(chrome_proxy_service.dart:222)
at <asynchronous gap>(async)
```

**Comment by christopherfujino** [comment]
cc @elliette

**Comment by elliette** [comment]
FYI @annagrin as well

**Comment by elliette** [comment]
This should have been fixed by https://github.com/dart-lang/webdev/pull/1938, which is included in DWDS v.18.0.0: https://github.com/dart-lang/webdev/releases/tag/dwds-v18.0.0

Currently 18.0.0 is on head: https://github.com/flutter/flutter/pull/121569

@christopherfujino does this need a cherry-pick or is it alright to roll out through the normal process?

**Comment by christopherfujino** [comment]
> This should have been fixed by [dart-lang/webdev#1938](https://github.com/dart-lang/webdev/pull/1938), which is included in DWDS v.18.0.0: https://github.com/dart-lang/webdev/releases/tag/dwds-v18.0.0
> 
> Currently 18.0.0 is on head: #121569
> 
> @christopherfujino does this need a cherry-pick or is it alright to roll out through the normal process?

regular rolling SGTM. I will close this as fixed on master. Thanks!
